### DISCUSSION TITLE
Color Codes Hex Values

### DISCUSSION THREAD

**Discussion by lparker00** [netbox-community/netbox#11948] [Help Wanted!] [answered]
I have imported rear and front ports with color values (how I set it up to correlate with fiber colors), and it works but the hex values I have for the colors, most of them, don't correspond with the default or drop down list in netbox so if I make an edit to the individual port, the color disappears. I was wondering if somebody had a list of the colors codes within Netbox. Thanks

**Comment by candlerb** [comment, marked as answer]
https://github.com/netbox-community/netbox/blob/v3.4.5/netbox/utilities/choices.py#L100-L127
